components/rich_text_editor/extensions/emoji/EmojiList.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="d-p4 d-bar8 d-bgc-contrast d-fs-200 d-truncate">
+    <template v-if="items.length">
+      <button
```

Comment:
should we use dt-list-item?

Reply:
used! still debating if we need to use the icon on the left slot or on default slot as it is putting too much padding.

Reply:
i actually removed the left slot and put it all on default. seems better that way.